app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, foreign_key: :post_id
+  has_many :likes
+
+  def recent_post
+    posts.order('created_at Desc').limit(3)
+  end
+
```

Comment:
- [ ] As the previous reviewer suggested, this method should return the 5 most recent comments for a given post, not 3.